homeassistant/components/analytics_insights/config_flow.py

```diff
@@ -53,10 +53,25 @@ async def async_step_user(
     ) -> FlowResult:
         """Handle the initial step."""
         self._async_abort_entries_match()
-        if user_input:
-            return self.async_create_entry(
-                title="Home Assistant Analytics Insights", data={}, options=user_input
-            )
+        errors: dict[str, str] = {}
+        if user_input is not None:
+            if not user_input.get(CONF_TRACKED_INTEGRATIONS) and not user_input.get(
+                CONF_TRACKED_CUSTOM_INTEGRATIONS
+            ):
+                errors["base"] = "no_integrations_selected"
+            else:
+                return self.async_create_entry(
+                    title="Home Assistant Analytics Insights",
+                    data={},
+                    options={
```

Comment:
If you add a `, default=[]` to the Schema keys, you can keep this as `options=user_input` right?

Reply:
no clue, I haven't tested that yet. This isn't an easy option for the options flow tho